h/models/user.py

```diff
@@ -59,27 +107,30 @@ def __getitem__(self, username):
 
 class User(Base):
     __tablename__ = 'user'
-    __table_args__ = (
-        sa.UniqueConstraint('email', 'authority'),
-        sa.UniqueConstraint('uid', 'authority'),
-        sa.UniqueConstraint('username', 'authority'),
-    )
+
+    @declared_attr
+    def __table_args__(cls):
```

Comment:
This has to become a `declared_attr`, so that the `ix__user__user` index can reference the `username` column.